website/docs/docs/collaborate/git/resolve-merge-conflicts.md

```diff
@@ -1,70 +1,66 @@
+- A 2-split editor view &mdash; The left view includes your code changes and is read-only. The right view includes the additional changes, allows you to edit and marks the conflict with some flags:
 
-You will then see that the git action bar shows `commit and resolve...` instead of `commit` indicating that the Cloud IDE has detected some conflicts that need to be addressed. You should also see the conflict section marked with some flags:
 ```
 <<<<<< HEAD
     your current code
 ======
     conflicting code
 >>>>>> (some branch identifier)
 ```
+- File and path are colored in red in the **File Explorer**, with a warning icon to highlight files that you need to resolve
+- File name is colored in red in the **Changes** section, with a warning icon
+- If you press commit without resolving the conflict, the Cloud IDE will prompt a pop up box with a list which files need to be resolved
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-ide/conflict-section.png" title="Conflicting section that needs resolution will be highlighted"/>
-
-<Lightbox src="/img/docs/dbt-cloud/cloud-ide/file-highlight.png" title="File and path are colored in red with a warning sign to highlight files that need to be resolved"/>
-
-<Lightbox src="/img/docs/dbt-cloud/cloud-ide/commit-warning.png" title="Pressing commit without resolving the conflict will also list which files need to be addressed"/>
-
-:::info Encountering Merge Conflicts
-
-You could run into this merge conflict in 2 possible ways:
-- Pulling changes from your main branch when someone else has merged a conflicting change
-- Committing your changes to the same branch when someone else has already committed their change first (this scenario)
-
-The way to resolve either scenario will be exactly the same!
-
-:::
-
-## Resolving conflicts
-
-<Lightbox src="/img/docs/dbt-cloud/cloud-ide/resolve-section.png" title="Choosing lines to keep"/>
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/merge-conflict.jpg" title="Conflicting section that needs resolution will be highlighted"/>
 
-1. Choose which lines of code you'd like to preserve. Delete the rest, make sure to also delete the special flags that highlight the merge conflict.
-2. Press save! You will notice the line highlights disappearing, returning to a plain white background, which is a good sign that you've resolved the conflict successfully!
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/commit-without-resolve.jpg" title="Pop up box when you commit without resolving the conflict"/>
 
-Repeat this process for every file that has a merge conflict.
 
-## Completing the process
+## Resolve merge conflicts
+You can seamlessly resolve merge conflicts that involve competing line changes in the Cloud IDE.
 
-When you've resolved all the merge conflicts, the last step would be to commit the changes you've made!
+1. In the Cloud IDE, you can edit the right-side view and choose which lines of code you'd like to preserve and delete the rest. 
```

Comment:
This is true, and I think it's sufficient to keep this as in because the nuances are kind of confusing.

But TECHNICALLY, for a conflict file
- if you open it from the diff view (from the files listed under Changes), note that the file name shows something like *model.sql (last commit)* then it is fully read-only
- If you open it from the file tree, then that file is editable in the right side -- tab name is *model.sql (conflict)*

Reply:
@bossnunta thanks so much for clarifying that and I've added that clarification for the user so hopefully it'll convey right. 

[deploy staging link](https://deploy-preview-2441--docs-getdbt-com.netlify.app/docs/collaborate/git/resolve-merge-conflicts#resolving-conflicts)